src/driver/connection.ts

```diff
@@ -93,6 +91,14 @@ export class Connection extends MongooseConnection {
         });
     }
 
+    async listDatabases(): Promise<{ databases: string[] }> {
+        return executeOperation(async () => {
+            await this._waitForClient();
+            const { status } = await this.client.findNamespaces();
```

Comment:
If the `response` from the Data API has `errors`, the `executeCommandWithUrl` function throws `StargateServerError`. Do we need a test case for that to check if the error is handled gracefully?

Reply:
@kathirsvn is there a way to get `findNamespaces()` to return a server error? Based on the [implementation](https://github.com/stargate/data-api/blob/0c3859a5e9e4605c42b1f75610842ee74e23dfcf/src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindNamespacesOperations.java#L21) and API I don't see a good way to get `findNamespaces()` to error out. I could sinon.stub(), but that seems a bit contrived.

Reply:
Got it, it is not thrown by the API, functionally for any input. Even Invalid token returns 401 and not the error body I think.